modules/statistics/php/statistics_site.class.inc

```diff
@@ -40,12 +40,20 @@ class Statistics_Site extends \NDB_Menu
      */
     function _hasAccess(\User $user) : bool
     {
-        $centerID = htmlspecialchars($_REQUEST['CenterID']);
```

Comment:
Sanitization not necessary here because the value is not passed to the front-end.